src/olympia/search/tests/test_search_ranking.py

```diff
@@ -484,8 +484,11 @@ def setUpTestData(cls):
     def test_scenario_tab_center_redux(self):
         self._check_scenario('tab center redux', (
             ['Tab Center Redux', 69.5371],
-            ['Tab Mix Plus', 0.06526235],
-            ['Redux DevTools', 0.044507127],
+            # Those used to be found but we now require all terms to be present
+            # through operator: and on the fuzzy name query (and they have
+            # nothing else to match).
+            # ['Tab Mix Plus', 0.06526235],
+            # ['Redux DevTools', 0.044507127],
```

Comment:
What would now happen when searching for `tab center`?  Would it match `Tab Center Redux` still? What about searching for `tab center pro` (or similar?)

Reply:
`tab center` does match `tab center redux` easily - both terms match, don't even need the fuzzy filter, it also matches the prefix rule and the match phrase rule, so tab center redux would be fairly high in the results. Locally it's the first result for me.

`tab center pro` wouldn't find it however, and that's the biggest change. Locally it doesn't find anything anymore, because that last bit, "pro", doesn't match anything that's also matching the first two. On prod, at the moment, it does find a bunch of add-ons, some relevant, some not. Those 3 issues I linked all complain about this behavior, so that's why I'm changing it here.

Do you think it's sensible ? You can test locally too, if you import prod add-ons using `manage.py fetch_prod_addons` (takes a while)

Reply:
As discussed on IRC, I've made the fuzzy query less rigid in https://github.com/mozilla/addons-server/pull/10004/commits/3a51d5cc99e15f399c6d2727d4e2989e4e7d4ec7 with a `minimum_should_match` instead of the `operator: and`.